Backend/dev/migrations/dynamic-offer-driver-app/0157-add-merchantId-in-registrationToken-table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE atlas_driver_offer_bpp.registration_token
+ADD COLUMN merchant_id TEXT NOT NULL DEFAULT 'favorit0-0000-0000-0000-00000favorit'
```

Comment:
Not all the token in prod environment would be of NAMMA_YATRI. Write a DB migration where merchant_id is filled from person table using `entity_id` of registration_token.

Reply:
Done